packages/client/lib/config.ts

```diff
@@ -17,6 +17,11 @@ export enum DataDirectory {
   Meta = 'meta',
 }
 
+export enum SyncMode {
+  full = 'full',
+  light = 'light',
```

Comment:
I think we should use PascalCase for enum keys here as we do in other parts of the monorepo (lowercase or camelCase for the values is fine)

```suggestion
  Full = 'full',
  Light = 'light',
```

Also, what do you think about moving this enum definition to `lib/types.ts`?

Reply:
makes sense :slightly_smiling_face: :+1:

Reply:
actually we can keep it in this file, makes sense here too 👍